example2/android/app/build.gradle

```diff
@@ -26,7 +26,7 @@ apply plugin: 'kotlin-android'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 33
```

Comment:
This we might have to lower down, if it creates any issues. Please keep a watch on this.